backend/app/celery/celery_app.py

```
@@ -1,8 +1,43 @@
+    "app.services.doctransform.job",
+    "app.services.collections.create_collection",
+    "app.services.collections.delete_collection",
+    "app.services.stt_evaluations.batch_job",
+    "app.services.tts_evaluations.batch_job",
+    "app.services.tts_evaluations.batch_result_processing",
+    "app.services.stt_evaluations.metric_job",
+]
+
+
+@worker_process_init.connect
+def warmup_job_modules(sender, **kwargs: object) -> None:
```

Ruff (0.15.6)

[warning] Unused function argument: `sender`

(ARG001)